uiowa/uiowa
# Counselors map fix

Resolves #7489 





# How to test


`ddev blt frontend` and check that there are no errors (if you want to check the output, the admissions_core and pharmacy_core build processes shouldn't include the babel es6 step)

```
ddev blt ds --site=admissions.uiowa.edu && ddev drush @admissions.local uli /counselors
```
Click on a state, see that the counselors list below updates
Click on a different state, and see that the counselors list updates again

```
ddev blt ds --site=pharmacy.uiowa.edu && ddev drush @pharmacy.local uli /pharmd/dual_cert/palliative/certificate_graduates
```
Click on a state, see that the grads list updates
Click on a different state, and see that the grads list updates again

Code review of `docroot/sites/admissions.uiowa.edu/modules/admissions_core/admissions_core.libraries.yml` updates for other es6 -> js transitions and/or testing the aos-kiosk and slideshow features, and that the js properly loads for each
